hack/install-operator.sh

```diff
@@ -6,7 +6,11 @@ BASEPATH=$( dirname $SELF )
 
 NAMESPACE=${1:-kubevirt}
 
-REGISTRY=$(minishift openshift registry)
+LAST_TAG=""
+if [ "${CI}" != "true" ] || [ "${TRAVIS}" != "true" ]; then
+	# TODO: consume releases, not tags
+	LAST_TAG=$( curl -s https://api.github.com/repos/MarSik/kubevirt-ssp-operator/tags | jq -r '.[0].name' )
```

Comment:
Note you are using simple curl here as well. This will be affected by the api throttling as the environment variable is not automatically passed in. You need to send a token explicitely: https://developer.github.com/v3/auth/

Reply:
On the other hand, this specific line is not going to be called from CI. But developers from the same office might still hit this (too many people with the same IP)

Reply:
@MarSik ack. instructions are indeed simple (https://developer.github.com/v3/auth/#via-oauth-tokens), but I wonder which pair user/token should i use. Mine? Yours? something else?

Reply:
I would use the GITHUB_TOKEN environment variable. We can then generate a proper token and add it to Travis.

Or GITHUB_TOKEN_API with fallback to GITHUB_TOKEN to allow for separate release (with permissions) and read only API (no extra permissions) tokens.

Reply:
OK, so https://github.com/fromanirh/traviskube/pull/1 is a good direction :) Let's continue here the discussion, then I'll update this PR accordingly

Reply:
Fixed in my traviskube fork.